README.md

```diff
@@ -16,9 +16,6 @@ transformato can be easily installed from conda-forge:
 ```
 conda install transformato -c conda-forge
 ```
```

Comment:
We don't need this information anymore since there was a big update for `parmed`